assets/extensions/card.js

```diff
@@ -7,11 +7,13 @@ import classnames from 'classnames';
  * WordPress dependencies
  */
 import { __ } from '@wordpress/i18n';
+import { useSelect } from '@wordpress/data';
 
 /**
  * Internal dependencies
  */
 import ExtensionActions, { getExtensionActions } from './extension-actions';
+import { EXTENSIONS_STORE } from './store';
```

Comment:
To bring everything together, I think it would be nice to move fetching the extension data to this store too. And probably even the page layout.

Reply:
Yep agreed! Probably should be done as part of another PR though, right?